.devcontainer/host_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+# Prepare to install packages
+sudo apt update
+
+# Install the prerequisites
+sudo apt install -y --no-install-recommends wget sudo xz-utils nano bc git python3 bison flex make cppcheck gdb
```

Comment:
One thing I added here was `gdb`, as I was trying to debug some [behavior I find odd in how `osquery` applies the `DISTINCT` constraint](https://github.com/NickBorgers/osquery-container-dependency-scanning/blob/master/README.md).